src/config.h.in

```diff
@@ -0,0 +1,4 @@
+#if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
```

Comment:
Done. Shall I delete commits related to configure.h.in, rebase and force
push now?

On Thu, Aug 11, 2016 at 7:12 PM, James Aylett <notifications@github.com>
wrote:

> In src/config.h.in
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74423492>:
>
> > @@ -0,0 +1,4 @@
> > +#if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
>
> Probably worth putting a quick comment at the top of this. In particular,
> I think you should note that this isn't automatically generated (some
> autotools projects have config.h.in generated automatically from
> configure.ac).
>
> You actually don't need it to be .in, thinking about it. You aren't using
> any @@…@@ replacements, so it'll probably just work as a normal header
> file. You probably also want #include "xapian.h" at the top, so they can
> be included in either order in source files.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/ef9626298d83c1842f1d45f0a724bb461f304a7e#r74423492>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNokJnrWc81J3BYIN3WFDCkj1myFzks5qeyapgaJpZM4Jf4Xo>
> .
>